masterfiles/analyze.py

```diff
@@ -0,0 +1,54 @@
+# TODO merge this with ENT-12099 branch cfbs analyze.py
+import os
```

Comment:
Please put these files inside the `cfbs` package is installed as one self contained package. (`./cfbs/masterfiles/analyze.py`, `./cfbs/masterfiles/download_all_versions.py`, etc.)

Reply:
Done in https://github.com/cfengine/cfbs/pull/208/commits/3f0f1ed42b9971a87d0bf816fe3e9ed416fd7371